### DISCUSSION TITLE
how to use pipe in node-fetch

### DISCUSSION THREAD

**Discussion by code-masala** [node-fetch/node-fetch#1390] [Q&A]
```js
fetch(url, {
		method: "POST",
		body:obj
	})
		.then((res) => res.json())
		.pipe(res);
```

**Comment by code-masala** [comment]
this is giving error @Richienb

**Comment by jimmywarting** [comment]
- pipe can only be used on the `res.body`
- json can not be streamable (for that you need another package like: https://www.npmjs.com/package/JSONStream)
either you want to do:
```js
fetch(url, {
  method: "POST",
  body:obj
})
  .then((res) => res.json())
  .then(json => {
    // ...
  });
```
or
```js
fetch(url, {
  method: "POST",
  body:obj
})
  .then((res) => {
    res.body.pipe(dest)
  })

**Comment by lafkpages** [comment]
Not sure if this is the same as native Fetch in Node but when trying to use Fetch to download a file, I get the following `TypeError: resp.body.pipe is not a function`

**Reply by zzBrunoBrito** [reply]
Same here, any updates?
